### PR TITLE
[#6784] ysql: use all rather than 0.0.0.0/0, ::0/0 in hba conf

### PR DESCRIPTION
**Pull request by pratikSethi**
Replaced the occurrences of 0.0.0.0/0 and ::0/0 for ipv4 and ipv6 addressing with **all** in hba.conf

### CODE REVIEW

File: src/yb/yql/pgwrapper/pg_wrapper.cc [hunk 1]
```diff
@@ -246,8 +244,8 @@ Result<string> WritePgHbaConfig(const PgProcessConf& conf) {
 
   // Enforce a default hba configuration so users don't lock themselves out.
   if (lines.empty()) {
-    lines.push_back("host all all 0.0.0.0/0 trust");
-    lines.push_back("host all all ::0/0 trust");
+    lines.push_back("host all all all trust");
+    lines.push_back("host all all all trust");
```
Comment (iSignal) [inline src/yb/yql/pgwrapper/pg_wrapper.cc:248]: we don't need both lines now

File: src/yb/yql/pgwrapper/pg_wrapper.cc [hunk 2]
```diff
@@ -232,10 +232,8 @@ Result<string> WritePgHbaConfig(const PgProcessConf& conf) {
     const auto host_type =  conf.enable_tls ? "hostssl" : "host";
     const auto auth_method = FLAGS_ysql_enable_auth ? (conf.enable_tls ? "md5 clientcert=1" : "md5")
                                                     : "cert";
-
-    for (const auto addr : {"0.0.0.0/0", "::0/0"}) {
-      lines.push_back(Format("$0 all all $1 $2", host_type, addr, auth_method));
-    }
+    lines.push_back(Format("$0 all all all $1", host_type, auth_method));
+    lines.push_back(Format("$0 all all all $1", host_type, auth_method));
```
Comment (iSignal) [inline src/yb/yql/pgwrapper/pg_wrapper.cc:236]: we don't need both lines now

**Review by iSignal — CHANGES_REQUESTED** [review]

File: src/yb/yql/pgwrapper/pg_wrapper.cc (hunk 1 above)
Comment (pratikSethi) [reply src/yb/yql/pgwrapper/pg_wrapper.cc]: You are right, I have updated my PR.

File: src/yb/yql/pgwrapper/pg_wrapper.cc (hunk 2 above)
Comment (pratikSethi) [reply src/yb/yql/pgwrapper/pg_wrapper.cc]: Change incorporated, thanks!

**Review by iSignal — APPROVED** [review]

**Pull request merged**